lib/assets/javascripts/utils/paginable.js

```diff
@@ -4,14 +4,16 @@ export const paginableSelector = '.paginable';
 
 $(() => {
   const onAjaxSuccessHandler = (e, data) => {
-    $(e.target).closest(paginableSelector).html($(data));
+    $(e.target).closest(paginableSelector).replaceWith($(data));
   };
   $('.paginable-search form[data-remote="true"]').on('ajax:before', e => isValidText($(e.target).find('input[name="search"]').val()));
   // Event listener for Ajax success event captured in response to a paginable link clicked or
   // search form submitted. Note the presence of a selector for on (e.g. a[data-remote="true"])
   // so that descendant elements from .paginable-results that are added in future are also
   // automatically handled.
-  $(paginableSelector).on('ajax:success', 'form.paginable-action[data-remote="true"], a.paginable-action[data-remote="true"]', onAjaxSuccessHandler);
+  $('body').on('ajax:success',
+    'form.paginable-action[data-remote="true"], a.paginable-action[data-remote="true"]',
```

Comment:
We should also check pages where there are multiple paginable tables (e.g. My Dashboard) to make sure that firing the success handlers for both tables doesn't cause any issues

Reply:
Good example of something to check durring UAT. Do we have a process for flagging things like this to @sjDCC and @stephaniesimms to ensure they get checked in UAT?